_pages/qa/when-can-I-take-leave-according-to-the-family-and-medical-leave-act.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: "q-and-a"
+permalink: "/when-can-I-take-leave-according-to-the-family-and-medical-leave-act/"
+title: When am I allowed to take leave according to the Family and Medical Leave Act?
+tags: fmla, family and medical leave act, maternity leave, paternity leave, parental leave, leave 
+---
+# {{ page.title }}
+
+
+An employee is entitled to taking FMLA when they have been an employee of the federal government for at least 12 months. Note: An employee who has transferred from another agency can use their time to count for 12-month restriction.
+
+The Department of Labor has a [detailed guide](http://www.dol.gov/whd/fmla/) to FMLA, including this [FAQ](http://www.dol.gov/whd/fmla/fmla-faqs.htm) which goes into extensive detail.
```

Comment:
This bit is duplicated from `/fmla-family-and-medical-leave-act/`. Perhaps replace with: `Read the [Family and Medical Leave Act page]({{ site.baseurl }}/fmla-family-and-medical-leave-act/) for more information.`